packages/global-cli/README.md

```diff
@@ -1,126 +1,27 @@
-## Running CLI with local modifications
+## React Native Global CLI
 
-React Native is distributed as two npm packages, `react-native-cli` and `react-native`. The first one is a lightweight package that should be installed globally (`npm install -g react-native-cli`), while the second one contains the actual React Native framework code and is installed locally into your project when you run `react-native init`.
+`react-native-cli` is a command line tool for initializing React Native projects.
 
-Because `react-native init` calls `npm install react-native`, simply linking your local github clone into npm is not enough to test local changes.
+## Usage
 
-### Introducing Sinopia
+We recommend using `react-native-cli` with `npx` so you don't need to pollute your global packages scope.
```

Comment:
TBH I’m personally again `npx` because of security reasons, maybe write the command to install globally too?

Reply:
What kind of security reasons? I don't think I'm aware :o

Reply:
Well basically npx (AFAIK) is able to run *anything* (= a pack from the web that contains whatever) on your machine as basically an admin... which is a big no no in my books ;)

Reply:
Oh, I guess that's when you use global node version. Will adjust that.

Reply:
👍